pkgs/by-name/pi/picom-pijulius/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  picom,
+  lib,
+  fetchFromGitHub,
+  pcre
+}:
+picom.overrideAttrs (
+  finalAttrs: previousAttrs: {
+    pname = "picom-pijulius";
+    version = "8.2-2024-05-17-unstable";
+
+    src = fetchFromGitHub {
+      owner = "pijulius";
+      repo = "picom";
+      rev = "e7b14886ae644aaa657383f7c4f44be7797fd5f6";
+      hash = "sha256-YQVp5HicO+jbvCYSY+hjDTnXCU6aS3aCvbux6NFcJ/Y=";
+    };
+
+    buildInputs = previousAttrs.buildInputs ++ [ pcre ];
+
+    meta = with lib; {
```

Comment:
Move the `with lib;` down and then `meta = previousAttrs.meta // { `

Reply:
`previousAttrs.meta` would be wrong, as `meta.position` for example would point at the original package. better to `inherit` only the specific attrs that are needed.